docs/MigrationGuide.md

```diff
@@ -29,7 +29,7 @@ To see the biggest differences please consult the [changelog](Changelog.md).
 * [#4745](https://github.com/TouK/nussknacker/pull/4745) Added method `ScenarioBuilder` to create fragments with specified input node id instead of taking a default 
   from fragment id
 * [#4745](https://github.com/TouK/nussknacker/pull/4745) Add more errors for scenario and node id validation and change names, messages of existing ones
-* [#4928](https://github.com/TouK/nussknacker/pull/4928) [#5028](https://github.com/TouK/nussknacker/pull/5028) `Validator.isValid` method 
+* [#4539](https://github.com/TouK/nussknacker/pull/4928) [#5028](https://github.com/TouK/nussknacker/pull/5028) `Validator.isValid` method 
```

Comment:
Looks like some merge conflict that was not resolved properly...

Reply:
It was late hour ;)